apps/web/src/pages/farms/index.tsx

```diff
@@ -56,11 +53,13 @@ const FarmsPage = () => {
       {chosenFarmsMemoized?.map((farm) => {
         if (farm.version === 2) {
           const farmCakePerSecondNum =
-            farm.poolWeight && cakePerSecond ? (Number(farm.poolWeight) * Number(cakePerBlock)) / 1e18 / 3 : 0
+            farm.poolWeight && cakePerSecond
+              ? (Number(farm.poolWeight) * Number(ethers.utils.parseEther(cakePerBlock.toString()))) / 1e18 / 3
+              : 0
 
           const farmCakePerSecond =
             farmCakePerSecondNum === 0
-              ? '-'
+              ? '0'
               : farmCakePerSecondNum < 0.000001
```

Comment:
also i decided to use 0 instead of the - here for when either cakePerSec or a multiplier is 0 (perhaps based on the farm info) as when using the app i felt that when the UI showed -, it meade me feel as if something was broken, rather than the - being the default. I feel that showing 0 is more truthful and a better indication, but this is just me. can revert if there was a specific reason for using the -

Reply:
Let's use `-` in this case, normally it won't be `0` unless the farm is finished